README.md

```diff
@@ -1,2 +1,39 @@
+
 # dev-welcome-pr
+
 SEF Community Contributors Welcome Pull Requests
+
+[![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/sef-global/sef-site/issues)
+
+
+# How to make your welcome PR
+
+
+## 1. Create MarkDown file inside the folder _profiles 
+
+* Fork this repository, create a new markdown file inside the `_profiles` folder and name it with your username. It should look something like this `_profiles/<YOUR-USERNAME>.md`. Ex.
```

Comment:
We should specify that it's the sef hive user name we ask for here right?

Reply:
Yeah, @Gravewalker666 . It will be better to use hive user names while creating profiles. I will update `your username` to `Your sef hive username`.  Is that okay?